src/DataViews/LiveFunctionsDataViewTest.cpp

```diff
@@ -915,4 +915,21 @@ TEST_F(LiveFunctionsDataViewTest,
   EXPECT_CALL(app_, ShowHistogram(nullptr, "", std::optional<ScopeId>{})).Times(1);
 
   view_.UpdateHistogramWithScopeIds({kNonDynamicallyInstrumentedFunctionId});
+}
+
+TEST_F(LiveFunctionsDataViewTest, OnDataChangedUsesScopeStatsCollectionUpdates) {
```

Comment:
I think, this should ideally be split into two test cases -- test initialisation and test addition of a row.